dask/dataframe/groupby.py

```diff
@@ -2223,9 +2222,9 @@ def aggregate(self, arg, split_every=None, split_out=1, shuffle=None):
         )
 
     @_aggregate_docstring(based_on="pd.core.groupby.DataFrameGroupBy.agg")
-    def agg(self, arg, split_every=None, split_out=1, shuffle=None):
+    def agg(self, arg, split_every=None, split_out=1, **kwargs):
```

Comment:
Using kwargs here should be fine since we are just passing everything through to `self.aggregate` (which will raise if unexpected kwargs are detected)

Reply:
I prefer to avoid `**kwargs` unless absolutely necessary (especially when they mask self-documentation for function signatures). Can you say a bit more about why we need this?

Reply:
> especially when they mask self-documentation for function signatures

Good call, I changed this a bit to avoid kwargs

>Can you say a bit more about why we need this?

Attempted to explain here: https://github.com/dask/dask/pull/9403#issuecomment-1219720591